frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -151,28 +195,41 @@ export default ({
         [PROPOSAL_GENERIC]: () => L('TODO: Change [generic] from [current] to [new-value]', {})
       }[this.proposalType]()
     },
+    humanDate () {
+      const date = new Date(this.proposal.meta.createdDate)
+      const offset = date.getTimezoneOffset()
+      const minutes = date.getMinutes()
+      date.setMinutes(minutes + offset)
+      const locale = navigator.languages !== undefined ? navigator.languages[0] : navigator.language
+
+      return date.toLocaleDateString(locale, {
+        year: 'numeric', month: 'long', day: 'numeric'
+      })
```

Comment:
This code will give the wrong date display, in fact a PR was just accepted that fixed this, and this would re-introduce the bug. Have a look here in ProposalBox.vue: https://github.com/okTurtles/group-income/pull/1331/files#diff-55810e398eeec154b5fc2e69e1d0f145b06292609f25792c0d360e9a4264c310L76